src/compass/utils/helpers.py

```diff
@@ -253,25 +256,29 @@ def get_burst_bbox(burst_id, burst_db_file=None, burst_db_conn=None):
         burst_db_conn = sqlite3.connect(burst_db_file)
     burst_db_conn.row_factory = sqlite3.Row  # return rows as dicts
 
-    burst_ids = [burst_id] if isinstance(burst_id, str) else burst_id
+    burst_ids = [burst_id] if isinstance(burst_id, str) else \
+        ','.join(burst_id)
 
-    results = []
-    query = "SELECT epsg, xmin, ymin, xmax, ymax FROM burst_id_map WHERE burst_id_jpl = ?"
-    for bid in burst_ids:
-        cur = burst_db_conn.execute(query, (bid,))
-        results.append(cur.fetchone())
+    query = "SELECT epsg, xmin, ymin, xmax, ymax burst_id_jpl FROM burst_id_map WHERE burst_id_jpl = ?"
+    cur = burst_db_conn.execute(query, (burst_ids,))
```

Comment:
This gives me an error: 
```
In [45]: compass.utils.helpers.burst_bbox_from_db('t087_185678_iw1', burst_db_file=db_file)
---------------------------------------------------------------------------
InterfaceError                            Traceback (most recent call last)
Input In [45], in <cell line: 1>()
----> 1 compass.utils.helpers.burst_bbox_from_db('t087_185678_iw1', burst_db_file=db_file)

File /u/aurora-r0/staniewi/repos/COMPASS/src/compass/utils/helpers.py:263, in burst_bbox_from_db(burst_id, burst_db_file, burst_db_conn)
    259 burst_ids = [burst_id] if isinstance(burst_id, str) else \
    260     ','.join(burst_id)
    262 query = "SELECT epsg, xmin, ymin, xmax, ymax burst_id_jpl FROM burst_id_map WHERE burst_id_jpl = ?"
--> 263 cur = burst_db_conn.execute(query, (burst_ids,))
    264 results = cur.fetchall()
    266 if not results:

InterfaceError: Error binding parameter 0 - probably unsupported type.
> /u/aurora-r0/staniewi/repos/COMPASS/src/compass/utils/helpers.py(263)burst_bbox_from_db()
    261
    262     query = "SELECT epsg, xmin, ymin, xmax, ymax burst_id_jpl FROM burst_id_map WHERE burst_id_jpl = ?"
--> 263     cur = burst_db_conn.execute(query, (burst_ids,))
    264     results = cur.fetchall()
    265

ipdb> p burst_ids
['t087_185678_iw1']
```

i think if you make `burst_ids` a list, you just want to do 

```suggestion
    cur = burst_db_conn.execute(query, burst_ids)
```